sematic/api/endpoints/resolutions.py

```diff
@@ -406,3 +407,40 @@ def _update_resolution_user(
     resolution.settings_env_vars[var_key] = user.api_key
 
     return resolution, True
+
+
+@sematic_api.route("/api/v1/resolutions/with_orphaned_jobs", methods=["GET"])
+@authenticate
+def get_orphaned_resolution_job_identifiers_endpoint(
+    user: Optional[User],
+) -> flask.Response:
+    run_ids = get_resolutions_with_orphaned_jobs()
```

Comment:
```suggestion
    resolution_ids = get_resolutions_with_orphaned_jobs()
```
Same thing but more consistent 🤷

Reply:
done